src/tlo/methods/contraception.py

```diff
@@ -645,24 +645,89 @@ def select_contraceptive_following_birth(self, mother_id, mother_age):
-        _cons_codes['pill'] = get_items_from_pkg('Pill')
-        _cons_codes['male_condom'] = get_items_from_pkg('Male condom')
-        _cons_codes['other_modern'] = get_items_from_pkg('Female Condom')
-        # NB. The consumable female condom is used for the contraceptive state of "other_modern method"
-        _cons_codes['IUD'] = get_items_from_pkg('IUD')
-        _cons_codes['injections'] = get_items_from_pkg('Injectable')
-        _cons_codes['implant'] = get_items_from_pkg('Implant')
-        _cons_codes['female_sterilization'] = get_items_from_pkg('Female sterilization')
+        # # items for each method that requires an HSI to switch to
+        # TODO: @TimH Pills are alternatives, we weight their use by the probabilities (as approximation of use in 80:20
+        #  cases, for each case 0.8 and 0.2 proportion of units is used). Is that okay, or do we want to choose for each
+        #  case only one type of pills? Total usage should be approximately same.
+        # TODO: @Sakshi what are the numbers of pills in monthly packets for both following items?
+        _cons_codes['pill'] =\
+            {get_item_code("Levonorgestrel 0.0375 mg, cycle"): 28*3.75*0.2,  # progesterone-only pills used in 20% cases
+             get_item_code("Levonorgestrel 0.15 mg + Ethinyl estradiol 30 mcg (Microgynon), cycle"): 21*3.75*0.8
+             # combined pills used in other 80% cases
+             }
+        _cons_codes['male_condom'] =\
+            {get_item_code("Condom, male"): 30}
+        _cons_codes['other_modern'] =\
+            {get_item_code("Female Condom_Each_CMST"): 30}
+        _cons_codes['IUD'] =\
+            {get_item_code("Glove disposable powdered latex medium_100_CMST"): 2,
+             get_item_code("IUD, Copper T-380A"): 1}
+        # TODO: originally used "Medroxyprogesterone acetate injection 150mg/mL, 1mL vial with 2ml syringe with 22g 0.7
+        #  X 25mm needle_each_CMST", which is not within the RF_Costing anymore, does it mean we need to add a syringe
+        #  and a needle?
+        _cons_codes['injections'] = \
+            {get_item_code("Depot-Medroxyprogesterone Acetate 150 mg - 3 monthly"): 1,
+             get_item_code("Glove disposable powdered latex medium_100_CMST"): 1,
+             get_item_code("Water for injection, 10ml_Each_CMST"): 1,
+             get_item_code("Povidone iodine, solution, 10 %, 5 ml per injection"): 5,
+             get_item_code("Gauze, swabs 8-ply 10cm x 10cm_100_CMST"): 1}
+        # TODO: trocar may be reusable (we used 0.1 units, indicating 10 uses on average) - need to find out for what
+        #  kind of trocar (reusable or not) we have the cost
+        # Jadelle & Implanon are alternatives, we weight their use by the probabilities (as approximation of use in
+        # 50:50 cases, for each case 0.5 and 0.5 proportion of units is used)
+        # TODO: @TimH Is that okay, or do we want to choose for each case only one type of implant? Total usage should
```

Comment:
As above, doing this way will require both items to be available in any checks on availability. So I would recommend choosing one here.